### ISSUE TITLE
Make the Homebrewery meet Web Content Accessibility Guidelines(WCAG)

### ISSUE THREAD

**Issue by 5e-Cleric** [naturalcrit/homebrewery#3032]
As the title says, i think it would be epic if we made the homebrewery more accessible to people with disabilities, the WCAG international standard provides the necessary information on how to do so.

As per 2023, the most recent stable version is WCAG 2, so i suggest that standard is the one used for this issue.

**Comment by G-Ambatte** [comment]
For future reference - WCAG 2 guidelines: https://www.w3.org/TR/WCAG20/

**Comment by 5e-Cleric** [comment]
Adding here some issues that should be solved, related to this:

Editor:
- [ ] Add pointer cursor icon to the entire foldgutter element
- [ ] Add highlight to the foldgutter element
- [ ] Add `not-allowed` cursor icon to snippetbar functions if they are not available (brew history stuff)
- [ ] Remove draggability from thumbnail image preview
- [ ] Normalize text input look (border-radius, padding) in properties editor

Preview:
- [ ] Add `ew-resize` cursor to  toolbar zoom function
- [ ] Increase width of page number to at least 4 digits (4ch + padding)
- [ ] Add `not-allowed` cursor icon to disabled buttons

Navigation:
- [ ] Add `wait` cursor to saving element while saving
- [ ] Add hover effect to storage and new save button

Some of these can be done globally:

`*:disabled { cursor:not-allowed; }`

**Comment by dbolack-ab** [comment]
Is this being broken out of the uI update pR?
